docs/_guide/properties.md

```diff
@@ -104,15 +104,18 @@ Remember to call `super()` first in your constructor, or your element won't rend
 
 {% include project.html folder="properties/declare" openFile="my-element.js" %}
 
-### Declare properties with TypeScript decorators
+### Declare properties with decorators
 
-You can also declare properties with TypeScript decorators:
+You can also declare properties with decorators:
 
 ```js
 @property({type : String})  prop1 = 'Hello World';
 ```
 
-**Example: Declare properties with TypeScript decorators** 
+Decorators are a proposed JavaScript feature, so you'll need to use a transpiler like Babel or the TypeScript compiler to use decorators. 
+With Babel, you'll need to use the `@babel/plugin-proposal-decorators` plugin. Decorators are supported by default in TypeScript.
```

Comment:
Decorators aren't supported by default in TypeScript. You need `"experimentalDecorators": true` in .tsconfig. You _don't_ need `emitDecoratorMetadata` (and should ideally have that off as it bloats the output).

Reply:
TIL. Thanks!